### ISSUE TITLE
Error loading plugins  Kimai2 php 8.1.2

### ISSUE THREAD

**Issue by joefrancois** [mxgross/EasyBackupBundle#54]
Hello,

I get the following error when I try to load the plugin.
_________________
In PluginMetadata.php line 59:
                                                                                                
  Bundle "EasyBackupBundle" defines an invalid Kimai minimum version in extra.kimai.require. P  
  lease provide an integer as in Constants::VERSION_ID.      

_________________
my PHP build is:

PHP 8.1.2-1ubuntu2.10 (cli) (built: Jan 16 2023 15:19:49) (NTS)
Copyright (c) The PHP Group
Zend Engine v4.1.2, Copyright (c) Zend Technologies
    with Zend OPcache v8.1.2-1ubuntu2.10, Copyright (c), by Zend Technologies

Thank you and regards,
Joseph

**Comment by mxgross** [comment]
Hi,
which Kimai Version do you use and what version of EasyBackup Plugin?

**Comment by StefanGebhardt** [comment]
Same Problem for me. I have kimai2 2.0.12 and tried to install EasyBackup 1.4.2 via 

`git clone https://github.com/mxgross/EasyBackupBundle.git EasyBackupBundle`

Cloning worked without problems
```
Cloning into 'EasyBackupBundle'...
remote: Enumerating objects: 929, done.
remote: Counting objects: 100% (51/51), done.
remote: Compressing objects: 100% (43/43), done.
remote: Total 929 (delta 8), reused 21 (delta 6), pack-reused 878
Receiving objects: 100% (929/929), 691.37 KiB | 6.84 MiB/s, done.
Resolving deltas: 100% (478/478), done.
```
 but after

`bin/console kimai:reload --env=prod`

I got the error

```
In PluginMetadata.php line 59:
                                                                                                                                                     
  Bundle "EasyBackupBundle" defines an invalid Kimai minimum version in extra.kimai.require. Please provide an integer as in Constants::VERSION_ID.  
```

**Comment by mxgross** [comment]
Hi @StefanGebhardt 
in Kimai2 2.x.x and higher big changes where made. 
I began to transform also EasyBackup plugin to the new platform, but it is not yet on master branch.

You can checkout and test my branch "53-plugin-compatibility-for-kimai-200-beta-and-higher" to run the backup on Kimai2 2.x

**Comment by StefanGebhardt** [comment]
Maybe its a problem because of the provider all-inkl.
See: https://github.com/kimai/kimai/discussions/2859
I had the same problems and did the workaround. So only with this workaroud kimai2 works. Maybe it has also impacts on the installation of plugins. I tried now to install another plugin - same problem. And after installation kamai2 is not working any longer. Deleting the plugin directory - and it works again.

**Comment by mxgross** [comment]
If this was not a provider issue, it should be solved now to run Kimai 2 V 2.0.0 and higher with EasyBackup version 2.0.1 and higher. (branch called 'main' now)
